js/leaflet-idw.js

```diff
@@ -0,0 +1,354 @@
+/*
```

Comment:
Third-party libs should be included with https://git-scm.com/docs/git-submodule instead, unless we're making changes, which is generally a bad idea anyway.. (we should just submit PRs to the original repo in that case, which is easy cause it's on GitHub! we could even fork the repo into our org)